src/core/layout/qgslayoutexporter.cpp

```diff
@@ -532,10 +532,9 @@ QgsLayoutExporter::ExportResult QgsLayoutExporter::exportToPdf( const QString &f
   ExportResult result = printPrivate( printer, p, false, settings.dpi, settings.rasterizeWholeImage );
   p.end();
 
-  const bool shouldGeoreference = mLayout->pageCollection()->pageCount() == 1;
```

Comment:
What about the page count restriction?

Reply:
@nyalldawson , I've exported a georeferenced mutli-page PDF here and it worked just fine. Is that restriction based on an obsolete bottleneck?

Reply:
It's more about it potentially being misleading. (Eg multi page export with many maps)

Reply:
@nyalldawson , OK, what about georeferencing multi-page PDFs only if the mLayout->referenceMap()->page() == 0 ?

Reply:
No strong feelings :)

Reply:
@nyalldawson , good, I've implemented that logic :) If a PDF can't be georeferenced (no reference map present or reference map isn't on the first page), the export dialog will reflect that by graying out the option:
![Screenshot from 2019-08-12 16-00-36](https://user-images.githubusercontent.com/1728657/62854973-1aa07200-bd1b-11e9-8ae2-567bed59489b.png)

Users will not always know what they are exporting beyond what's visible on their PDF viewers ;)